lunes_cms/cms/admins/discipline_admin.py

```diff
@@ -206,6 +206,7 @@ def get_queryset(self, request):
             ),
         )
 
+    # pylint: disable=arguments-differ
     def get_form(self, request, obj=None, **kwargs):
```

Comment:
I guess this is a false positiv 🤔 I couldn't find 5th parameter in `get_form` function, though the pylint insists `Number of parameters was 5 in 'ModelAdmin.get_form' and is now 4 in overriding`

Reply:
Good question :thinking: Maybe it would be good if a second person could do a quick check here?

Reply:
@timobrembeck @ulliholtgrave 
Do you think too this is a false positive?

Reply:
I think this refers to https://github.com/django/django/blob/2bcf555763f3d4fa19c135ac895e5d16e9d7141a/django/contrib/admin/options.py#L790